common/beerocks/bcl/include/bcl/network/network_utils.h

```diff
@@ -94,6 +94,9 @@ class network_utils {
     static int get_iface_info(network_utils::iface_info &info, const std::string &iface_name);
     static bool get_raw_iface_info(const std::string &iface_name, raw_iface_info &info);
 
+    static std::shared_ptr<std::unordered_map<std::string, std::string>>
+    get_arp_table(bool mac_as_key = true);
```

Comment:
I don't think it is that important and I prefer to keep it as it for now.
If more people will share your opinion, I might change it.

Reply:
It would actually be a lot better to have a class to represent an ARP table, with methods to look up by MAC and by IP.

However, there's a trivial resolution to Ran's comment: we don't need lookup by IP, and I don't expect we will ever need it because prplmesh doesn't work with IP addresses at all. So simply remove the boolean argument.